code/src/main/java/com/onlinecontacttracing/storage/data-storage/NegativeUser.java

```diff
@@ -30,8 +37,16 @@ public String getUserEmail() {
     return userEmail;
   }
 
+  public long getFirstLoginInSeconds() {
+    return firstLoginSeconds;
+  }
+
+  public long getLastLoginInSeconds() {
+    return lastLoginSeconds;
+  }
+  
   @Override
   public String toString() {
-    return "Negative User- ID:" + userId + ", email:" + userEmail;
+    return "Negative User- ID: " + userId + ", email: " + userEmail;
```

Comment:
nit: Probably should be "User-ID" not "User- ID"

Also, do you know about String.format?

Reply:
String.format("Negative User-ID: %s, email: %s,", userId, userEmail); 

Might be easier.

Reply:
I meant User- ID: _____, email: ______

Like the - refers to ID and email if that makes sense

Reply:
So are you saying to replace it with String.format? Or to keep it in mind for the future?

Reply:
So you're saying you meant it to be "User- ID:______, (User) Email: _____" where the second User is implied by the "- "?

That's confusing. Just say User-ID and User-Email if that is your intention. 

Also, keep it in mind for the future. I don't care if you do it here, but String.format tends to be way more readable than what you're doing here. It's also slightly faster I think (String concatenation is expensive in Java). Though that's less of an issue.